src/properties/degrees.c

```diff
@@ -536,6 +536,58 @@ igraph_error_t igraph_degree_correlation_vector(
     return IGRAPH_SUCCESS;
 }
 
+igraph_error_t igraph_strength_all(const igraph_t *graph, igraph_vector_t *res,
```

Comment:
```suggestion
static igraph_error_t igraph_strength_all(const igraph_t *graph, igraph_vector_t *res,
```

This should be static since it's only used within this single translation unit.

Reply:
Can we also rename it to `igraph_i_strength_all()` to make it clear that it's internal (just by looking at the name of the function)?

Reply:
Yes, I was going to do that. I'm push some changes and will merge soon.